package.json

```diff
@@ -87,7 +87,7 @@
     "gatsby-transformer-remark": "^2.3.12",
     "gatsby-transformer-sharp": "^2.1.17",
     "gatsby-transformer-yaml": "^2.1.11",
-    "gh-pages": "2.0.1",
+    "gh-pages": "^2.0.1",
```

Comment:
This needs to be this exactly `2.0.1` because newer ones don't work to deploy to our `gh-pages` branch :(